app/dashboard/page.tsx

```diff
@@ -0,0 +1,291 @@
+"use client";
+
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useState,
+} from "react";
+import styles from "@styles/dashboard.module.css";
+import { useAccount, useStarkName } from "@starknet-react/core";
+import { usePathname, useRouter } from "next/navigation";
+import { useDebounce } from "@hooks/useDebounce";
+import { StarknetIdJsContext } from "@context/StarknetIdJsProvider";
+import Blur from "@components/shapes/blur";
+import DashboardSkeleton from "@components/skeletons/dashboardSkeleton";
+import { isHexString, minifyAddress } from "@utils/stringService";
+import useCreationDate from "@hooks/useCreationDate";
+import { utils } from "starknetid.js";
+import { decimalToHex, hexToDecimal } from "@utils/feltService";
+import ProfileCard from "@components/UI/profileCard/profileCard";
+import { LeaderboardTopperParams, fetchLeaderboardToppers, getBoostById, getBoosts, getCompletedQuests, getQuestById } from "@services/apiService";
+import { calculatePercentile } from "@utils/numberService";
+import { getDomainFromAddress } from "@utils/domainService";
+import BoostCard from "@components/quest-boost/boostCard";
+import { QuestDocument } from "../../types/backTypes";
+
+
+type DashboardProps = {
+  params: {
```

Comment:
i think this is not a valid way to declare page props. You might have to check the vercel docs to get the correct structure here.